Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:7.3-cli
```

Comment:
Osobne bych preferoval to postavit nad Alpine pokud vylozene neni potreba Debian.

Reply:
Tohle bylo asi nejrychlejší na zprovoznění, ale klidně to můžem postavit na něčem menším.